lib/Parser/perrors.h

```diff
@@ -6,85 +6,85 @@
-LSC_ERROR_MSG( 1064, ERRMissingCurlyBrace      , "Closing curly brace ('}') expected.")
-LSC_ERROR_MSG( 1065, ERRRestLastArg, "The rest parameter must be the last parameter in a formals list.")
-LSC_ERROR_MSG( 1066, ERRRestWithDefault, "The rest parameter cannot have a default initializer.")
-LSC_ERROR_MSG( 1067, ERRUnexpectedEllipsis, "Unexpected ... operator")
+LSC_ERROR_MSG(1063, ERRInvalidCodePoint      , "Invalid codepoint value in the escape sequence.")
+LSC_ERROR_MSG(1064, ERRMissingCurlyBrace      , "Closing curly brace ('}') expected.")
+LSC_ERROR_MSG(1065, ERRRestLastArg, "The rest parameter must be the last parameter in a formals list.")
+LSC_ERROR_MSG(1066, ERRRestWithDefault, "The rest parameter cannot have a default initializer.")
+LSC_ERROR_MSG(1067, ERRUnexpectedEllipsis, "Unexpected ... operator")
 
-LSC_ERROR_MSG( 1068, ERRDestructInit, "Destructuring declarations must have an initializer")
-LSC_ERROR_MSG( 1069, ERRDestructRestLast, "Destructuring rest variables must be in the last position of the expression")
-LSC_ERROR_MSG( 1070, ERRUnexpectedDefault, "Unexpected default initializer")
-LSC_ERROR_MSG( 1071, ERRDestructNoOper, "Unexpected operator in destructuring expression")
-LSC_ERROR_MSG( 1072, ERRDestructIDRef, "Destructuring expressions can only have identifier references")
+LSC_ERROR_MSG(1068, ERRDestructInit, "Destructuring declarations must have an initializer")
+LSC_ERROR_MSG(1069, ERRDestructRestLast, "Destructuring rest variables must be in the last position of the expression")
+LSC_ERROR_MSG(1070, ERRUnexpectedDefault, "Unexpected default initializer")
+LSC_ERROR_MSG(1071, ERRDestructNoOper, "Unexpected operator in destructuring expression")
+LSC_ERROR_MSG(1072, ERRDestructIDRef, "Destructuring expressions can only have identifier references")
 
-LSC_ERROR_MSG( 1073, ERRYieldInTryCatchOrFinally, "'yield' expressions are not allowed in 'try', 'catch', or 'finally' blocks")
-LSC_ERROR_MSG( 1074, ERRConstructorCannotBeGenerator, "Class constructor may not be a generator")
-LSC_ERROR_MSG( 1075, ERRInvalidAssignmentTarget, "Invalid destructuring assignment target")
-LSC_ERROR_MSG( 1076, ERRFormalSame, "Duplicate formal parameter names not allowed in this context")
-LSC_ERROR_MSG( 1077, ERRDestructNotInit, "Destructuring declarations cannot have an initializer")
-// 1078 -- removed
+LSC_ERROR_MSG(1073, ERRYieldInTryCatchOrFinally, "'yield' expressions are not allowed in 'try', 'catch', or 'finally' blocks")
+LSC_ERROR_MSG(1074, ERRConstructorCannotBeGenerator, "Class constructor may not be a generator")
+LSC_ERROR_MSG(1075, ERRInvalidAssignmentTarget, "Invalid destructuring assignment target")
+LSC_ERROR_MSG(1076, ERRFormalSame, "Duplicate formal parameter names not allowed in this context")
+LSC_ERROR_MSG(1077, ERRDestructNotInit, "Destructuring declarations cannot have an initializer")
+LSC_ERROR_MSG(1078, ERRCoalesce, "Coalescing operator '\?\?' not permitted unparenthesized inside '||' or '&&' expressions")
```

Comment:
New error message for the new syntax error. The changes above in this file are just white space.